Kitodo-ImageManagementModule/src/main/java/org/kitodo/imagemanagementmodule/ConvertRunner.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.imagemanagementmodule;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.NoSuchElementException;
+import java.util.Random;
+
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.ExecuteWatchdog;
+import org.apache.commons.exec.Executor;
+import org.apache.commons.exec.PumpStreamHandler;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.im4java.core.IMOperation;
+import org.kitodo.config.Config;
+
+/**
+ * Executes the {@code convert} command.
+ */
+class ConvertRunner {
+    private static final Logger logger = LogManager.getLogger(ConvertRunner.class);
+
+    /**
+     * {@code convert} command without path.
+     */
+    private static final String CONVERT_COMMAND = "convert";
+
+    /**
+     * Randomness generator used to distribute the requests evenly on several
+     * configured SSH hosts.
+     */
+    private static final Random RANDOMNESS_GENERATOR = new Random();
+
+    /**
+     * Default timeout of two hours.
+     */
+    private static final int TWO_HOURS = 7200;
```

Comment:
You should rename `TWO_HOURS` to like `DEFAULT_TIMEOUT` this as on any changes of the value `7200` the naming would be confusing.